ports/osgearth/portfile.cmake

```
@@ -2,59 +2,33 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO gwaldron/osgearth
     REF "osgearth-${VERSION}"
-    SHA512 2f764eb1fff21cff57a04eceb73643b372c8b70899114c88cdf9928d525517214959745cb70b99ad3d1def946bfb3f58f16b4969ee4117b7563398f2410ee3e2
+    SHA512 4a2b80c907ebf2b56966598f9e134ad910d3271757496fb1d906cc413eb2ad09da366a96635f0195696efe16ef1a649e13b6ec1d901a39ced0465be797f14221
     HEAD_REF master
-    PATCHES
-        remove-lerc-gltf.patch
-        install-plugins.patch
+    PATCHES patch-vcpkg-port-on.patch
 )
 
-if("tools" IN_LIST FEATURES)
-	message(STATUS "Downloading submodules")
-	# Download submodules from github manually since vpckg doesn't support submodules natively.
-	# IMGUI
-	#osgEarth is currently using imgui docking branch for osgearth_imgui example
-	vcpkg_from_github(
-		OUT_SOURCE_PATH IMGUI_SOURCE_PATH
-		REPO ocornut/imgui
-		REF cab7edd135fb8a02b3552e9abe4c312d595e8777 #docking branch
-		SHA512 26dfe94793bcc7b041c723cfbf2033c32e5050d87b99856746f9f3e7f562db15b9432bf92747db7823acbc6e366dbcb023653692bb5336ce65a98483c4d8232a
-		HEAD_REF master
-	)
-
-	# Remove exisiting folder in case it was not cleaned
-	file(REMOVE_RECURSE "${SOURCE_PATH}/src/third_party/imgui")
-	# Copy the submodules to the right place
-	file(COPY "${IMGUI_SOURCE_PATH}/" DESTINATION "${SOURCE_PATH}/src/third_party/imgui")
-endif()
-
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" BUILD_SHARED)
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
-        tools       OSGEARTH_BUILD_TOOLS
-        blend2d     WITH_BLEND2D
+        tools OSGEARTH_BUILD_TOOLS
```

Nitpick: Can we leave it as is for now?
```suggestion
        tools       OSGEARTH_BUILD_TOOLS
```
I see that there are no other options to turn off now. But IMO osgearth is large enough for users to desire some control.

---

"blend2d" should not be a user option. The only reason it was ever an option was because there were some compatibility problems between the two libraries in the past so we did this to get around that. Those issues are OBE so we want to get rid of the option.

I agree that osgEarth is large and can be componentized to some extent -- we will look at that for a future release!